.github/workflows/CI.yml

```diff
@@ -716,19 +716,31 @@ jobs:
           name: sunshine-macports
           path: artifacts/
 
-      - name: Fix screen capture permissions
-        if: ${{ matrix.os_version != 12 }}  # macOS-12 is okay
-        # can be removed if the following is fixed in the runner image
-        # https://github.com/actions/runner-images/issues/9529
-        # https://github.com/actions/runner-images/pull/9530
+      - name: Fix permissions
```

Comment:
Did the changes made here actually help?

Reply:
Here I'm not sure, but they will be useful to write tests in https://github.com/LizardByte/Sunshine/pull/2508